### DISCUSSION TITLE
how do dislikes of videos uploaded after google removed the dislike button are gathered?

### DISCUSSION THREAD

**Discussion by nimrodbens** [Anarios/return-youtube-dislike#639] [Q&A] [answered]
basically title.

**Comment by cyrildtm** [comment]
https://github.com/Anarios/return-youtube-dislike/blob/main/Docs/FAQ.md
https://blog.youtube/news-and-events/update-to-youtube/

**Comment by sy-b** [comment, marked as answer]
$$ \textup{RYD Dislike Count} = \left( \frac{\textup{RYD Users Like Count}}{\textup{RYD Users Dislike Count}} \right) \times \textup{Public Like Count} $$


For more info:

[![IReturn YouTube Dislike Explained](https://yt-embed.herokuapp.com/embed?v=GSmmtv-0yYQ)](https://www.youtube.com/watch?v=GSmmtv-0yYQ)

Formula mentioned at [00:38](https://youtu.be/GSmmtv-0yYQ?t=38)

**Reply by cyrildtm** [reply]
this and the formula could be included in FAQ
